docs/content/content-blocks.md

```diff
@@ -79,22 +130,12 @@ My content
 `````
 
 (admonitions:colons)=
-### New style admonitions
+### Markdown-friendly directives with `:::`
 
 The admonition syntax above utilises the general [directives syntax](content:myst/directives).
-This has the advantage of making it consistent with every other directive.
-However, a big disadvantage is that, when working in any standard Markdown editor (or with the Jupyter Notebook interface),
-the text they contain will not render nicely as standard Markdown (for Markdown previews).
-
-By enabling extended syntax in your `_config.yml`, you will gain access to an alternative syntax for admonitions:
-
-```yaml
-parse:
-  myst_extended_syntax: true
-```
+However, if you're using an interface that does not support {term}`MyST Markdown`, it will render as a "raw" code block.
```

Comment:
@choldgraf does jupytext treat the as `code` blocks? They remain contained in the markdown right and will be shown literally? Perhaps the use of `code block` has confused me.

Reply:
I think I know what you mean here -- perhaps "raw literal block"?